Hardware/Star64.md

```diff
@@ -0,0 +1,34 @@
+---
+riscv_hardware: true
+cmake_plat: star64
+xcompiler_arg: -DRISCV64=1
+platform: Star64
+arch: RV64GBC, RV64IMAC, RV32IMAFBC
+virtualization: "No"
+iommu: "No"
+simulation_target: false
+Status: "Unverified"
+Contrib: "Community"
+Maintained: "seL4 Foundation"
+soc: StarFive JH7110
+cpu:  U74-MC, E24
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Star64
+
+The Star64 is a RISC-V SBC by Pine64 based on the StarFive JH7110 SoC. Check
+[here](https://wiki.pine64.org/wiki/STAR64) for details.
+
+The StarFive JH7110 SoC is comprised of the SiFive U74-MC (four 64-bit U74
+cores and one 64-bit S7 core) and one SiFive E24 32-bit core. Note that by
```

Comment:
```suggestion
cores and one 64-bit S7 core) and one SiFive E24 32-bit (RV32IMAFBC) core. Note that by
```

Reply:
What about not dropping it and instead saying that seL4 only runs on the U74 cores since the other cores lack S-Mode?